concourse/pipelines/default/deploy-parameters.yml

```diff
@@ -1,2 +1,3 @@
 workspace: default
 govuk_infrastructure_branch: main
+skip_db_migrations: true
```

Comment:
I would recommend putting this variable in a separate file. Since we want db migrations to run in the test environment, but not in the integration environment.

Then in the `set_pipeline` step I think you can specify this file:

```yaml
      var_files:
        - govuk-infrastructure/concourse/pipelines/deploy-parameters.yml
        - govuk-infrastructure/concourse/parameters/((govuk_environment))/parameters.yml
```

Reply:
So changing from being workspace specific, to being environment specific?

Reply:
I like this idea!